src/LiveComponent/src/Attribute/LiveProp.php

```diff
@@ -58,6 +58,13 @@ final class LiveProp
      */
     private null|string|array $onUpdated;
 
+    /**
+     * @var bool
+     *
+     * Tells if this property should be bound to the URL
+     */
+    private bool $queryMapping;
```

Comment:
I know you and @smnandre discussed this change... but I did like `url` more. A `url: true` that, potentially later, could be changed to a `bool|UrlMapping` type with some new `UrlMapping` object. Then the user could have `url: new UrlMapping(..)`.

Though, I am ok with keeping the "query mapping" or "url mapping" terminology deeper in the code.

Reply:
Fixed!